app/templates/gentelella/admin/event/tickets/access_codes_create.html

```diff
@@ -227,9 +227,8 @@
 
         var host = (document.URL.split("//")[1].split("/"))[0];
         $("#code").change(function(){
-            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?code="+$(this).val());
+            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?access_code="+$(this).val());
```

Comment:
and i dont think it is going to affect previous working flow we now just need to update the Discount_code_generation in the source